catroid/src/main/java/org/catrobat/catroid/content/bricks/HideTextBrick.java

```diff
@@ -183,6 +187,19 @@ public View getPrototypeView(Context context) {
 		return null;
 	}
 
+	void setUserVariableName(UserVariable userVariable) {
+		userVariableName = Constants.NO_VARIABLE_SELECTED;
+		try {
+			userVariableName = userVariable.getName();
+		} catch (NullPointerException e) {
+			Log.d(TAG, "Nothing selected yet.");
+		}
```

Comment:
Ah, these new methods aren't used at all. Remove them.

About `public String userVariableName;`... Please do some research how to handle removed members with XStream.